crates/types/src/configuration.rs

```diff
@@ -4,26 +4,11 @@ use std::{path::PathBuf, time::Duration};
 use nalgebra::{Matrix3, Point2, Point3, Vector2, Vector3, Vector4};
```

Comment:
Why not remove the entire file?

Reply:
This is not possible?! Or is it? What do you imagine for higher level configuration structs?

Reply:
Ah, I see. Then these types need to stay in the types crate. But would it make sense to put them in separate files? The name `configuration.rs` sounds like a grouping of types that not necessarily belong together.

Reply:
I disagree, they belong together. All these types are configuration parameters to their respective nodes.

Reply:
I disagree with your disagreement, because your view is to group them by implementation details and I would rather like to have them grouped by domain correspondence. These types are part of the robotic domain and it should not be important that they are used in configuration.

Reply:
Let's get this merged now. Added this to this PR's description. Resolving this discussion now.